README.md

```diff
@@ -25,6 +25,18 @@ Make sure you have GCC-4.6, G++-4.6, and GFortran-4.6 installed. For
 a full list of external dependencies on Ubuntu you can look at our
 [Travis configuration](https://github.com/graalvm/sulong/blob/master/.travis.yml).
 
+On the Mac you can use Homebrew:
+
+    brew install gcc
```

Comment:
Unfortunately there are no files with `fortran` in the name at all installed by GCC 4.8 :(